### ISSUE TITLE
Plugin does not display on iPad

### ISSUE THREAD

**Issue by sta55en** [jasonmamy/cordova-wheel-selector-plugin#55]
When I use this plugin on any iPad, from iOS 11 up to the latest release that I updated to today; the plugin is called successfully through Capacitor/Cordova, but it fails to display the selector on the screen. I have placed breakpoints in the native code and the code is definitely executed.

Any ideas?

Also - when I look at the release history on this repo, I see only 1 release, but if I look at NPM, I see many releases with more current versions. Why is that? Has the official repo moved?

**Comment by jasonmamy** [comment]
I don't have any iOS devices so won't be of much help. I thought that I
remember someone had a similar issue of calling through Cordova and the
plugin not showing up. I dug thru the issues but didn't see anything. Maybe
someone using it on iOS will chime in. I always used the plugin from Ionic,
so never actually called it directly from Cordova.

For the releases, yea I was lazy and didn't create a github release, just
pushed the releases into npm. Good to point out, I'll create github
releases when pushing to npm in the future. If you look at the history of
the package.json file, you'll see where the releases were spun from:

https://github.com/jasonmamy/cordova-wheel-selector-plugin/commits/master/package.json

**Comment by ashishvadhwa** [comment]
i have same issue can you please help me out i have same issue plugin is not display in ipad in iphone it works perfect with capacitor. I have test with cordova it also works fine in ipad but with capacitor it not works.

**Comment by jasonmamy** [comment]
You mention that the plugin will not display in ipad? Is it that it works
in ipad/iphone but not in capacitor?

**Comment by ashishvadhwa** [comment]
@jasonmamy yes right with capacitor plugin not display in ipad. In Iphone it is work perfect with capacitor only issue with ipad.

**Comment by jasonmamy** [comment]
Given capacitor is newer technology and the plug-in was written for Cordova
it may not work in capacitor yet.
